admin-docs/metadata-extraction.md

```diff
@@ -0,0 +1,101 @@
+
+### Limitations
+
+While the machine learning models used for metadata extraction are powerful, they do have limitations. The accuracy of extraction depends on how similar your documents are to those the models were trained on. If your documents have unusual formats, inconsistent language, or unique content, the system may require human intervention to ensure correct details are extracted. Additionally, complex or heavily unstructured documents, like lengthy legal narratives, may require more manual review after extraction. Therefore, while this tool is effective, it’s best used alongside human validation, especially for sensitive or detailed work.
+
+### Prerequisites
+
+Before using the metadata extraction tool, there are a few prerequisites to consider. These include access to the tool, the type of data it works with, and your familiarity with the document content you want to extract.
+
+- **Access**: This tool is in the beta phase and isn’t activated automatically for all Uwazi instances. Contact us if you’d like to access it during this phase. Note that when activated, the tool is only accessible to users with the roles of admin or editor.
+
+- **Available data**: The tool currently works with PDF documents and can extract information to specific template properties. Supported properties include text, numeric, date, single select, multiple select, and relationship. Your instance should have appropriate templates with documents available for metadata extraction.
+
+- **Knowledge of document content**: It’s important to have a clear understanding of the kind of information you want to extract, such as specific properties or categories relevant to your documents. This preparation will help ensure a smooth and accurate extraction process.
+
+## How to use metadata extraction effectively
+
+This step-by-step guide will walk you through the process of extracting metadata from your documents. First, initiate the metadata extraction by selecting the relevant properties from the available templates. The tool will then generate and present the extracted metadata as suggestions. Finally, review the suggestions, accepting those that are correct and refining any that need adjustments. Following these steps will help ensure that the extracted metadata is both accurate and relevant.
+
+### Creating a metadata extractor
+
+Step 1: Go to **Settings** ![Settings icon](images/cog-solid.png) and select **Metadata Extraction**.
+
+Step 2: Click on **Create Extractor**.
+
+Step 3: In the dialog window that appears, enter a name for the extractor in the input field.
+
+Step 4: From the list of available templates and properties, choose the property to be used for metadata extraction, then click **Next**.
```

Comment:
Maybe explain that only templates and properties that qualify for the extractors will appear

Reply:
Resolved with [0117d0b](https://github.com/huridocs/uwazi-documentation/pull/49/commits/0117d0b71e600075f49388686830dd5b54958f9f)